README.md

```diff
@@ -11,18 +11,21 @@ Please remember openSUSE is not SUSE, and neither this nor any other website wit
 
 ## How to build?
 
+Required software on openSUSE Hosts:  ruby3.2-devel, and gcc-c++
 ```bash
-bundle install --path vendor/bundle
-bundle exec jekyll build
+bundler.ruby3.2 add webrick
```

Comment:
You may as well just add it to the Gemfile here, no need to subject everyone to running this

Reply:
There you go.